package.json

```diff
@@ -50,6 +50,7 @@
     "clipboard-copy": "^4.0.1",
     "firebase": "^9.6.9",
     "fuse.js": "^6.5.3",
+    "html-truncate": "^1.2.2",
```

Comment:
This package doesn't have external dependency (https://github.com/huang47/nodejs-html-truncate/blob/master/package.json)

https://bundlephobia.com/package/html-truncate
<img width="1569" alt="image" src="https://user-images.githubusercontent.com/12745166/160087642-690d1819-12b3-4882-81da-b7ffeb593889.png">